src/styles/utils/mixins.scss

```diff
@@ -0,0 +1,13 @@
+@mixin onTablet {
+  @media (min-width: 576px) {
+    @content;
+  }
+}
+
+@mixin contentPaddingInline() {
+  padding-inline: 20px;
```

Comment:
Remove the parentheses after the mixin name since they are not necessary when there are no arguments.